app/editor/src/features/admin/sources/connections/Syndication.tsx

```diff
@@ -34,6 +43,14 @@ export const Syndication: React.FC = (props) => {
           setFieldValue('connection.import', e.currentTarget.checked);
         }}
       />
+      <FormikCheckbox
+        label="Fetch Content Body Separately"
+        name="connection.fetch-content"
+        tooltip="Whether content body is located remotely"
+        onChange={(e) => {
+          setFieldValue('connection.fetch-content', e.currentTarget.checked);
```

Comment:
Change name of property to `connection.fetchContent` so that it's a valid property name.